src/docs/patterns/navigation.mdx

```diff
@@ -1,70 +1,220 @@
+    />
+    <MaterialDesignDescription
+        title={'Navigation Drawer '}
+        url={'https://material.io/components/navigation-drawer/'}
+        style={{ marginRight: 0 }}
+    />
+    <Divider />
+</React.Fragment>
+
+# Drawer Navigation
+
+A navigation drawer is a panel that is attached on the side of your application. These typically contain a [List](/patterns/lists) of primary pages in your application for the user to access quickly.
+
+<ImageGrid images={[NavigationBreakdownImg]} regularWidth />
+
+Most drawers have four main sections:
+
+-   **Header**: contains the name of the company, name of the product or a logo
+-   **Subheader**: optional section that can be used to support custom content, such as filtering or search
+-   **Body**: contains a list of primary pages that users can navigate too
+-   **Footer**: contains company logo, version number and trademark information
+
+There are several different styles of drawer navigation, each with slightly different behavior.
+
+<DemoCard
+    repository={'Drawer'}
+    angular={'https://pxblue-components.github.io/angular/?path=/story/components-drawer--within-a-drawer-layout'}
+    react={'https://pxblue-components.github.io/react/?path=/story/components-drawer--in-drawer-layout'}
 />
 
-## Fixed Navigation
+## Permanent
 
-The most basic navigation panel is fixed in place and always visible on the left side of the application. Note: a fixed navigation panel is not suitable when going to smaller resolutions or mobile devices.
+A permanent drawer is always visible and always open. The permanent drawer is typically full-width, as opposed to the rail variant (see below). Use this variation if you have:
 
-## Collapsible Navigation
+-   five or more top-level pages
+-   a nested navigation hierarchy
+-   enough space to justify it (typically desktop-only applications)
 
-A more common implementation of the navigation panel is a collapsible panel that can be opened and closed via the menu icon in the top left corner of the application. This implementation can have two different variations: content-resize or content-cover.
+<ImageGrid images={[NavigationPermanentImg]} regularWidth />
 
-### Content-Resize / Content-Push
+## Rail
 
-<ImageGrid images={[PermanentImg]} />
+A navigation rail is a permanently collapsed drawer. The rail is able to show less information than a standard drawer, but has the advantage of a more compact, space-saving package. Use this variation if you have:
 
-The content-resize implementation does not cover any of the main application content. Rather, the navigation panel slides in and the main application container resizes to accommodate (the navigation panel and main panel are side by side instead of one on top of the other).
+-   fewer than seven top-level pages
+-   a flat navigation hierarchy
+-   limited space - recommended for HMI devices
 
-### Content-Cover
+<ImageGrid images={[NavigationRailImg]} regularWidth />
```

Comment:
![image](https://user-images.githubusercontent.com/8997218/94961683-1dcc6380-04c3-11eb-9f48-273f38282692.png)
avoid showing a menu icon here. The menu icon invokes a "drawer-opening" action, but a navigation rail aways stays collapsed.

Reply:
We can show two pics, one for the 72px rail, one for the 56px rail